### ISSUE TITLE
[Bug]: Area images do not display when using external URL

### ISSUE THREAD

**Issue by cerebrate** [junalmeida/homeassistant-minimalistic-area-card#32]
### What happened?

When accessing Home Assistant using its internal URL, the card correctly displays the area images. However, when accessing it from outside the local network (using its external URL, via a nginx reverse proxy), the card displays with a white background (as distinct from a broken-image marker) and the images are not visible.

(This only occurs when using the area images or the camera_image feature; it did not occur previously when specifying the image in the card configuration.)

Expected: images to display properly regardless of access URL.

I note, interestingly, that the HTML is different in each case. From outside the local network, the entire `<img>` tag for the card background image is missing.

### Card Version

1.0.6

### HomeAssistant Version

2022.12.8

### Relevant javascript log output

```shell
No relevant output was seen.
```

**Comment by junalmeida** [comment]
Thanks for digging and see the img image is missing, that's a good clue to start.

**Comment by junalmeida** [comment]
I see the card version you have is 1.0.6. Could you please try with latest version?

**Comment by cerebrate** [comment]
I may have misstated the card version last time, unfortunately, since I took the version number from the browser debug tools which I was looking at at the time; specifically the console output ```Minimalistic Area Card   1.0.6``` attributed to ```minimalistic-area-card.js:40```, per https://github.com/junalmeida/homeassistant-minimalistic-area-card/blob/main/src/minimalistic-area-card.ts#L17 , but since this is the same in the new version, 1.1.3 per HACS, I suspect the last version I had may actually have been 1.1.2.

In any case, the issue continues to appear in 1.1.3. To give more details this time, the HTML for the card element is this from the external URL:

```
<ha-card style="" tabindex="0" class="type-custom-minimalistic-area-card">
    <!--?lit$49252215317$-->
    <div class="box">
        <div class="card-header"><!--?lit$49252215317$-->Living Room</div>
        <div class="sensors">
            <!--?lit$49252215317$--><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$49252215317$-->
        <div class="state">
            <!--?lit$49252215317$-->
        </div>
        
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$49252215317$-->
        <div class="state">
            <!--?lit$49252215317$-->70 °F
        </div>
        
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$49252215317$-->
        <div class="state">
            <!--?lit$49252215317$-->28 %
        </div>
        
    </div>
    <!---->
        </div>
        <div class="buttons">
            <!--?lit$49252215317$-->
            <!--?lit$49252215317$--><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$49252215317$-->
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class="  ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$49252215317$-->
    </div>
    <!---->
        </div>
    </div>
<card-mod slot="none"><!---->
      <style>
        <!---->
      <!----></style>
    </card-mod></ha-card>
```

and this, from the internal:

```
<ha-card style="" tabindex="0" class="type-custom-minimalistic-area-card">
    <!--?lit$476583303$-->
    <!--?lit$476583303$--><div class="camera"><hui-image .width="100%"></hui-image></div>

    <div class="box">
        <div class="card-header"><!--?lit$476583303$-->Living Room</div>
        <div class="sensors">
            <!--?lit$476583303$--><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$476583303$-->
        <div class="state">
            <!--?lit$476583303$-->
        </div>
        
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$476583303$-->
        <div class="state">
            <!--?lit$476583303$-->70 °F
        </div>
        
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$476583303$-->
        <div class="state">
            <!--?lit$476583303$-->28 %
        </div>
        
    </div>
    <!---->
        </div>
        <div class="buttons">
            <!--?lit$476583303$-->
            <!--?lit$476583303$--><!---->
    <div class="wrapper">
        <ha-icon-button class=" state-on ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$476583303$-->
    </div>
    <!----><!---->
    <div class="wrapper">
        <ha-icon-button class="  ">
            <ha-state-icon></ha-state-icon>
        </ha-icon-button>
        <!--?lit$476583303$-->
    </div>
    <!---->
        </div>
    </div>
<card-mod slot="none"><!---->
      <style>
        <!---->
      <!----></style>
    </card-mod></ha-card>
```

Which shows in this case the missing camera DIV. Doing the same procedure with one of the non-camera cards yields the same effect only with the IMG tag in the opening

```
<ha-card style="" tabindex="0" class="type-custom-minimalistic-area-card">
    <!--?lit$1528716722$--><img src="https://jeeves.harmony.arkane-systems.lan/api/image/serve/f06020dfa5767a30e3cb3215d04d6135/512x512">
    <!--?lit$1528716722$-->
```

going missing.

**Comment by junalmeida** [comment]
Does your area cards work when accessing from both external and internal?  (from this url: https://your-url/config/areas/dashboard) 
The code tries to fetch the area settings, it seems that something is null in your case

**Comment by cerebrate** [comment]
I think I found the same issue from another direction, actually: initially, when I first started using the card (pre-1.1.0), I had configured explicit images for each, using the `image:` option, and if I'm right, the issue started with 1.1.0 _because_ I had initially configured no images for areas; I wasn't using the inbuilt HA area card, so there hadn't been a need.

It looks like when I, on noticing the relevant entry in the changelog last night changed from using explicit images to configuring the images in the area settings, it started working.

So it looks like the real issue is that explicit images don't work when there isn't also an area image configured? Not sure why it only shows up on the external URL, though.
